### PR TITLE
Implement React frontend + API in Django

### PR DESCRIPTION
**Pull request by ilya90906767**

### CODE REVIEW

**Comment by greenjune-ship-it** [comment]
### Restore `node_modules`

I think the whole `node_modules` content can be restored from the `frontend/package-lock.json` file.
We definitely don't want to include dependencies source code into our code base.
It is better to restore the environment once during the build of `frontend/Dockerfile` and then just reuse the container.

Do we have any cons for this?

File: django_app/django_app/settings.py [hunk 1]
```diff
@@ -31,18 +31,68 @@
...
 
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'rest_framework_simplejwt.authentication.JWTAuthentication',
+        
+    )
+}
+
+from datetime import timedelta
+...
```
Comment (greenjune-ship-it) [inline django_app/django_app/settings.py:57]: We don't need this line :)

File: docker-compose.yml [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+      - ./frontend:/app
+      # One-way volume to use node_modules from inside image
+      - /app/frontend/node_modules
+    ports:
+      - "3000:3000"
+    environment:
+      - NODE_ENV=development
+    depends_on:
+      - django-app
+    command: npm start
```
Comment (greenjune-ship-it) [inline docker-compose.yml:16]: Can we put it as `CMD ["npm", "start"]` in Dockerfile?
It's better to not overwhelm the compose file with details.

File: package.json
Comment (greenjune-ship-it) [inline package.json:1]: Do we need two separate `package.json` files? I think should be enough to have one in the `frontend` folder.

**Comment by ilya90906767** [comment]
> ### Restore `node_modules`
> I think the whole `node_modules` content can be restored from the `frontend/package-lock.json` file. We definitely don't want to include dependencies source code into our code base. It is better to restore the environment once during the build of `frontend/Dockerfile` and then just reuse the container.
> 
> Do we have any cons for this?

I deleted all useless .json files for npm. Now there are only 2 of them. Also i fixed Dockerfile

**Comment by greenjune-ship-it** [comment]
Duplicated by PR #3

**Pull request closed without merging**